tests/util/hash.py

```diff
@@ -8,9 +9,20 @@ def unordered(file_path):
     """
     file_hash = 1
     eighth_mersenne_prime = 2147483647
-    with open(file_path) as file:
+    with open(file_path, errors="ignore") as file:
         print("Hashing", file_path)
         for line in file:
             file_hash = (zlib.adler32(line.encode()) * file_hash) % eighth_mersenne_prime
     print("Hash is", file_hash)
     return file_hash
+
+
+def binary_hash(file_path):
+    """
+    Generates an MD5 hash of a binary file.
+    """
+    with open(file_path, "rb") as file:
+        print("Hashing", file_path)
+        file_hash = hashlib.md5(file.read()).hexdigest()
+    print("Hash is", file_hash)
```

Comment:
Fyi, the only reason the `print` statements were included in the `adler32` hash above is because I ported this over from a Jupyter notebook and missed that cleanup.

Not sure if print statements add any benefit here ¯\_(ツ)_/¯

Reply:
They generally help for manually invoking the hashes during testing (which is the only place hashes are being used), so I'll keep them in for now.

Reply:
👍